### DISCUSSION TITLE
Emacs with Ocaml

### DISCUSSION THREAD

**Discussion by JuzuSss** [3b1b/manim#2063] [Q&A]
Hello
I installed emacs in order to use caml on windows11 for my future projects. Except everything never goes as planned, after installing Ocaml, Emacs, Tuareg, Merlin ... I have a cmd page that opens when I open emacs: 

(process:17644): GLib-GIO-WARNING **: 22:24:10.172: Unexpectedly, UWP app `Clipchamp.Clipchamp_2.7.9.0_neutral__yxz26nhyzhsrt' (AUMId `Clipchamp.Clipchamp_yxz26nhyzhsrt!App') supports 41 extensions but has no verbs

(process:17644): GLib-GIO-WARNING **: 22:24:10.253: Unexpectedly, UWP app `Microsoft.ScreenSketch_11.2306.44.0_x64__8wekyb3d8bbwe' (AUMId `Microsoft.ScreenSketch_8wekyb3d8bbwe!App') supports 29 extensions but has no verbs

Can someone please help me?
(knowing that I've made sure I have everything installed)

**Comment by danielcranford** [comment]
You can suppress the error by redirecting stderr to a file or dropping it altogether, eg in cmd or powershell `emacs 2> errors.txt`

This seems to be an unresolved issue for emacs usage of manim on windows https://github.com/3b1b/manim/issues/1556
